pkgs/applications/version-management/git-lfs/default.nix

```diff
@@ -11,6 +11,13 @@ buildGoPackage rec {
     sha256 = "sha256-T4s/xnu5nL6dkEIo7xGywaE+EPH2OnzsaCF9OCGu7WQ=";
   };
 
+  patches = [
+    # patch git-lfs to install the git-lfs hook and configuration using the
+    # absolute path to the git-lfs command. Doing so makes the git-lfs
```

Comment:
Do those not path not be garbage collected at some point. This seems less optimal than the current state.

Reply:
hmm Good point. I should re-do the patch to offer a flag on `git lfs install` so it can be a controlled behavior. I'm going to also submit it upstream perhaps they would accept it and would relieve us from maintaining it.